lte/gateway/deploy/roles/magma_deb/tasks/main.yml

```diff
@@ -0,0 +1,161 @@
+  file:
+    src: "/home/vagrant/magma/lte/gateway/configs/{{ item }}"
+    dest: "/etc/magma/{{ item }}"
+    state: link
+    force: yes
+  loop:
+    - pipelined.yml
+    - sessiond.yml
+
+- name: Ensure changes in repository are used in services
+  file:
+    src: "/home/vagrant/magma/lte/gateway/configs/{{ item }}"
+    dest: "/etc/magma/{{ item }}"
+    state: link
+    force: yes
+  loop:
+    - templates/mme.conf.template
+    - gateway.mconfig
+
+- name: Clear existing service files
+  shell: rm -f /etc/systemd/system/magma*
+
+- name: Override production service configurations with test configurations
+  copy:
+    src: "/home/vagrant/magma/{{ item.src }}"
+    dest: "/etc/systemd/system/{{ item.dest }}"
+    remote_src: yes
+  with_items:
+    - { src: 'orc8r/tools/ansible/roles/gateway_services/files/magma.service', dest: 'magma@.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_connectiond.service', dest: 'magma@connectiond.service' }
+    - { src: 'orc8r/tools/ansible/roles/gateway_services/files/magma_control_proxy.service', dest: 'magma@control_proxy.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_ctraced.service', dest: 'magma@ctraced.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_dnsd.service', dest: 'magma@dnsd.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_envoy_controller.service', dest: 'magma@envoy_controller.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_eventd.service', dest: 'magma@eventd.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_liagentd.service', dest: 'magma@liagentd.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_lighttpd.service', dest: 'magma@lighttpd.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_magmad.service', dest: 'magma@magmad.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_mme.service', dest: 'magma@mme.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_mobilityd.service', dest: 'magma@mobilityd.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_pipelined.service', dest: 'magma@pipelined.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_redirectd.service', dest: 'magma@redirectd.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_redis.service', dest: 'magma@redis.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_sessiond.service', dest: 'magma@sessiond.service' }
+    - { src: 'orc8r/tools/ansible/roles/fluent_bit/files/magma_td-agent-bit.service', dest: 'magma@td-agent-bit.service' }
+    - { src: 'lte/gateway/deploy/roles/magma/files/systemd/magma_dp_envoy.service', dest: 'magma_dp@envoy.service' }
+
+- name: Reload magmad to apply service file changes
```

Comment:
This step is redundant if you are eventually rebooting the system.

Reply:
Thanks for spotting that, I've just removed it.